### PR TITLE
add ToolTip for AirMode

### PR DESCRIPTION
**Pull request by nerdCopter**
- https://discord.com/channels/868013470023548938/1019022739044057128/1293513438156296222
- modifies and re-uses `auxiliaryHelpMode_AIRMODE`
- link opens https://betaflight.com/docs/development/modes#airmode

image
image

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
image

**Comment by nerdCopter** [comment]
so many words to confuse the pilot :laughing: and use of "you" as the pilot.  oh i see that came from the .com, how about i link the .com?

**Comment by haslinghuis** [comment]
Perhaps:

_In standard mode, when roll, pitch, and yaw calculations saturate a motor, all motors are equally reduced. If a motor goes below minimum, it gets clipped off. For example, with throttle just above minimum and a quick roll is attempted, two motors can't go any lower, resulting in half the power (half of the PID gain). If inputs demand more than a 100% difference between high and low motors, the low motors get clipped, causing an imbalance by unevenly reducing the gain._

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
- i've taken it out of draft. 
- personally i don't like the wordiness -- I'd have preferred something much simpler such as `Normalizes motor output at low throttle to allow maintaining of the craft.`, but this is not my call.

**Comment by VitroidFPV** [comment]
Such a big pop-over element might cause UX and UI style issues, I think shortening it is more beneficial than having it be fully detailed. If links work in tooltip boxes, might be a good idea to link to the wiki for the full explanation

**Comment by nerdCopter** [comment]
thanks @VitroidFPV , I really agree, i think i'll revert to short text next work-week unless there are more comments to the contrary.

**Comment by nerdCopter** [comment]
@VitroidFPV @KarateBrot @mituritsyn  
re-used the first sentence, which allude "pid at min throttle" and linked the .com
thoughts?

**Comment by mituritsyn** [comment]
> thanks @VitroidFPV , I really agree, i think i'll revert to short text (+link) next work-week unless there are more comments to the contrary.

I personnaly vote for @ctzsnooze [formulation](https://discord.com/channels/868013470023548938/1019022739044057128/1293697224701644883):
Airmode' == the mixer behaviour where 'throttle is automatically raised, to maintain authority at low throttle. Just need to rephrase a bit

File: locales/en/messages.json [hunk 1]
```diff
@@ -2408,7 +2411,7 @@
         "description": "Help text to ANTIGRAVITY mode"
     },
     "auxiliaryHelpMode_AIRMODE": {
-        "message": "In the standard mixer / mode, when the roll, pitch and yaw gets calculated and saturates a motor, all motors will be reduced equally. When a motor goes below minimum it gets clipped off. Say you had your throttle just above minimum and tried to pull a quick roll - since two motors can't go any lower, you essentially get half the power (half of your PID gain). If your inputs would have asked for more than a 100% difference between the high and low motors, the low motors would get clipped, breaking the symmetry of the motor balance by unevenly reducing the gain",
+        "message": "AirMode allows for maintaining PID control at low throttle. For more information please see <a href=\"https://betaflight.com/docs/development/modes#airmode\" target=\"_blank\" rel=\"noopener noreferrer\">AirMode</a> on Betaflight.com.",
```
Comment (KarateBrot) [inline locales/en/messages.json:2414]:
```suggestion
        "message": "AirMode allows for maintaining drone stability at low throttle. For more information please see <a href=\"https://betaflight.com/docs/development/modes#airmode\" target=\"_blank\" rel=\"noopener noreferrer\">AirMode</a> on Betaflight.com.",
```

**Review by KarateBrot — APPROVED** [review]

**Comment by nerdCopter** [comment]
:(
i was still editing, but my internet provider is going hay-wire.  i was not
ready for merge.
i did not like the word `drone` as `locales/en/messages.json` only uses it
one other instance. and also wanted to account for Mike's preference.

**Comment by haslinghuis** [comment]
Apologies assumed this was ready after 24 hours cool of period.

**Comment by nerdCopter** [comment]
i try to stay offline on weekends. saw the comments this morning, but
internet intermittent.

```diff
index 94199765..dd4d487a 100755
--- a/locales/en/messages.json
+++ b/locales/en/messages.json
@@ -1234,7 +1234,8 @@
         "message": "Downlink:"
     },
     "featureNone": {
-        "message": "&lt;Select One&gt;"
+        "message": "&lt;Select One&gt;",
+        "description": "Don't translate!!!"
     },
     "featureRX_PPM": {
         "message": "PPM/CPPM (single wire)"
@@ -2411,7 +2412,7 @@
         "description": "Help text to ANTIGRAVITY mode"
     },
     "auxiliaryHelpMode_AIRMODE": {
-        "message": "AirMode allows for maintaining PID control at low
throttle. For more information please see <a href=\"
https://betaflight.com/docs/development/modes#airmode\" target=\"_blank\"
rel=\"noopener noreferrer\">AirMode</a> on Betaflight.com.",
+        "message": "AirMode is a motor mixer behavior that automatically
raises throttle during low-throttle input in order to maintain the craft's
authority. For more information please see <a href=\"
https://betaflight.com/docs/development/modes#airmode\" target=\"_blank\"
rel=\"noopener noreferrer\">AirMode</a> on Betaflight.com.",
         "description": "Help text to AIRMODE mode"
     },
     "auxiliaryHelpMode_BEEPER": {
```

either i can PR again, or just let it be.

**Pull request merged**